### ISSUE TITLE
when the scroller's height is less than the wrapper'height

### ISSUE THREAD

**Issue by yqz0203** [cubiq/iscroll#929]
when the scroller's height is less than the wrapper'height,i can't scroll anyway.

it cause a problem, I need pull to refresh the data, but I can't pull at all...

whether i should set a min-height to the scroller?

**Comment by swenyang** [comment]
I found a solution for this, just set the scroller's "min-height" property either in CSS(if the wrapper's height is static) or programmatically in JS(if the wrapper's height is dynamic). 
```js
var wrapper = document.getElementById("wrapper")
var scroller = document.getElementById("scroller")
scroller.style.minHeight = (wrapper.clientHeight + 1) + "px"
var myIScroll = new iScroll(wrapper, {/*your options*/})
```

**Comment by yqz0203** [comment]
@swenyang yes..I sovled it like you said, thank you for replying after such a long time. 😊

**Comment by kaansoral** [comment]
I had a solution is iscroll level, if the content height is lower, iScroll wasn't enabling scroll, you can modify that part of the code to make it always scroll

This solution is obviously simpler, if there are no side effects

**Comment by swenyang** [comment]
@kaansoral If your team is managing packages with npm or bower, modifying library code directly is not good. Also, could you show your code to tell which line to modify?

**Comment by kaansoral** [comment]
Indeed it's not good, I'm stuck at 5.1.3 for example, it was good until this package started being maintained again

Just checked the code, here is the modification:
```
/* REPLACE START: refresh */

		this.scrollerWidth	= this.scroller.offsetWidth;
		this.scrollerHeight	= this.scroller.offsetHeight;

		this.maxScrollX		= this.wrapperWidth - this.scrollerWidth;
		this.maxScrollY		= this.wrapperHeight - this.scrollerHeight;
		
		if(this.maxScrollY>=0) this.maxScrollY=-1; // #CUSTOM - forcing the scroll even if the height is low

/* REPLACE END: refresh */
```

**Comment by swenyang** [comment]
Thanks, basically it's working except that the scrollbar doesn't display correctly on scrolling..
